settings.gradle

```diff
@@ -36,6 +36,7 @@ includeWithFlags ':tomcat8.5',                          'java', 'publish', 'relo
 includeWithFlags ':zookeeper',                          'java', 'publish', 'relocate'
 includeWithFlags ':saml',                               'java', 'publish', 'relocate'
 includeWithFlags ':bucket4j',                           'java', 'publish', 'relocate'
+includeWithFlags ':annotation-processor',               'java', 'publish', 'relocate'
```

Comment:
How about publishing this artifact after we implement the logic in doc service which uses the processor?